app/lib/actions/katello/alternate_content_source/update.rb

```diff
@@ -19,14 +19,14 @@ def plan(acs, smart_proxies, products, acs_params)
           products_to_associate = []
           products_to_disassociate = []
 
-          if acs.simplified?
-            products = products.uniq
-            products_to_associate = products - acs.products
-            products_to_disassociate = acs.products - products
-            old_product_ids = acs.products.pluck(:id)
-            acs.products = products
-            acs.audit_updated_products(old_product_ids) unless products_to_associate.empty? && products_to_disassociate.empty?
-          end
+          products = products.uniq
```

Comment:
```suggestion
          products.uniq!
```

Reply:
Thank you

Reply:
Have to deny this change because it unfortunately breaks the program. Went down a bit of a rabbit hole trying to figure out why; If you have time for a meeting I can explain.

Reply:
No worries!